kinetic/distribution.yaml

```diff
@@ -15759,6 +15759,20 @@ repositories:
       url: https://github.com/NicksSimulationsROS/sync_params.git
       version: ros-kinetic
     status: developed
+  t_mapping:
+    doc:
+      type: git
+      url: https://github.com/mitre/t_mapping.git
```

Comment:
I have to say that the name `t_mapping` is too generic.  That is, it violates the first Global Rule here: https://www.ros.org/reps/rep-0144.html#global-rules .  Can we call it something more descriptive of what it does?